src/CMakeLists.txt

```diff
@@ -1,549 +1,534 @@
+    libraries/qkeysequencewidget/qkeysequencewidget/src/qkeysequencewidget.h
+    libraries/qkeysequencewidget/qkeysequencewidget/src/qkeysequencewidget.cpp
+    libraries/sonnet/src/core/backgroundchecker.cpp
+    libraries/sonnet/src/core/backgroundchecker.h
+    libraries/sonnet/src/core/backgroundchecker_p.h
+    libraries/sonnet/src/core/client.cpp
+    libraries/sonnet/src/core/client_p.h
+    libraries/sonnet/src/core/core_debug.cpp
+    libraries/sonnet/src/core/core_debug.h
+    libraries/sonnet/src/core/guesslanguage.cpp
+    libraries/sonnet/src/core/guesslanguage.h
+    libraries/sonnet/src/core/languagefilter.cpp
+    libraries/sonnet/src/core/languagefilter_p.h
+    libraries/sonnet/src/core/loader.cpp
+    libraries/sonnet/src/core/loader_p.h
+    libraries/sonnet/src/core/settings.cpp
+    libraries/sonnet/src/core/settings_p.h
+    libraries/sonnet/src/core/sonnetcore_export.h
+    libraries/sonnet/src/core/speller.cpp
+    libraries/sonnet/src/core/speller.h
+    libraries/sonnet/src/core/spellerplugin.cpp
+    libraries/sonnet/src/core/spellerplugin_p.h
+    libraries/sonnet/src/core/textbreaks.cpp
+    libraries/sonnet/src/core/textbreaks_p.h
+    libraries/sonnet/src/core/tokenizer.cpp
+    libraries/sonnet/src/core/tokenizer_p.h
+
+    threads/scriptthread.cpp
+    threads/scriptthread.h
+
+    widgets/qownnotesmarkdowntextedit.cpp
+    widgets/qownnotesmarkdowntextedit.h
+    widgets/navigationwidget.cpp
+    widgets/navigationwidget.h
+    widgets/notepreviewwidget.cpp
+    widgets/notepreviewwidget.h
+    widgets/combobox.cpp
+    widgets/combobox.h
+    widgets/graphicsview.cpp
+    widgets/graphicsview.h
+    widgets/scriptlistwidget.cpp
+    widgets/scriptlistwidget.h
+    widgets/notefolderlistwidget.cpp
+    widgets/notefolderlistwidget.h
+    widgets/fontcolorwidget.ui
+    widgets/fontcolorwidget.cpp
+    widgets/fontcolorwidget.h
+    widgets/logwidget.cpp
+    widgets/logwidget.h
+    widgets/logwidget.ui
+    widgets/label.cpp
+    widgets/label.h
+    widgets/lineedit.cpp
+    widgets/lineedit.h
+    widgets/scriptsettingwidget.cpp
+    widgets/scriptsettingwidget.h
+    widgets/scriptsettingwidget.ui
+    widgets/qtexteditsearchwidget.cpp
+    widgets/qtexteditsearchwidget.h
+    widgets/qtexteditsearchwidget.ui
+    widgets/notetreewidgetitem.cpp
+    widgets/notetreewidgetitem.h
+    widgets/notetreewidgetitem.ui
+    widgets/layoutwidget.cpp
+    widgets/layoutwidget.h
+    widgets/layoutwidget.ui
+    utils/misc.cpp
+    utils/misc.h
+    utils/git.cpp
+    utils/git.h
+    utils/gui.cpp
+    utils/gui.h
+    utils/schema.cpp
+    utils/schema.h
+    build_number.h
+    main.cpp
+    mainwindow.cpp
+    mainwindow.h
+    mainwindow.ui
+    release.h
+    version.h
+    helpers/fakevimproxy.h
+    helpers/fakevimproxy.cpp
+    services/websocketserverservice.cpp
+    services/websocketserverservice.h
+    entities/bookmark.h entities/bookmark.cpp)
+
+if(NOT APPLE)

```

Comment:
this is unnecessary now. All platforms use hunspell so you can remove this check

Reply:
I don't have Mac so I do not know.

Reply:
Haha, I dont have mac either. I wrote this back when i first integrated the spellchecker, didn't workout very well so we removed it. Seems like no one updated the CMake ;)